ebu_tt_live/bindings/test/test_times.py

```diff
@@ -0,0 +1,72 @@
+
+from unittest import TestCase
+from unittest import skip, SkipTest
+from datetime import timedelta
+from ebu_tt_live.bindings import ebuttdt
+from pyxb import SimpleTypeValueError, SimpleFacetValueError
+
+class TestTimecountTimingType(TestCase):
+
+    _test_cases = {
+        '1.3s': timedelta(seconds = 1, milliseconds = 300),
+        '1.03s': timedelta(seconds = 1, milliseconds = 30),
+        '1.30s': timedelta(seconds = 1, milliseconds = 300),
+        '1s': timedelta(seconds = 1),
+        '1.3m': timedelta(minutes = 1, seconds = 18),
+        '1.03m': timedelta(minutes = 1, seconds = 1, milliseconds = 800),
+        '1.30m': timedelta(minutes = 1, seconds = 18),
+        '1m': timedelta(minutes = 1),
+        '1.3h': timedelta(hours = 1, minutes = 18),
+        '1.03h': timedelta(hours = 1, minutes = 1, seconds = 48),
+        '1.30h': timedelta(hours = 1, minutes = 18),
+        '1h': timedelta(hours = 1),
+        '1.3ms': timedelta(milliseconds = 1, microseconds = 300),
+        '1.03ms': timedelta(milliseconds = 1, microseconds = 30),
+        '1.30ms': timedelta(milliseconds = 1, microseconds = 300),
+        '1ms': timedelta(milliseconds = 1),
+    }
```

Comment:
As up to three decimal places are supported, the tests should check the (corner) case of three decimal places, too.

Reply:
And maybe also the case of four decimal places...

Reply:
I've added a fix for up to 3 decimal places.

Reply:
The 3 decimal places constraint was not a spec requirement, and the new approach in 50ce306 means it is no longer needed. Nevertheless I have added tests for 3 decimal places and also in 50ce306 some tests for some 4 decimal place values.